CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Contribution Guidelines
+
+### Contributor Agreement
+
+To contribute source code to this repository, please read our [Contributor Agreement](http://www.mulesoft.org/legal/contributor-agreement.html), and then execute it by running [this notebook](https://api-notebook.anypoint.mulesoft.com/notebooks/#380297ed0e474010ff43) and following the instructions.
+
+## Setting-up this project locally
+
+### Installation
+```sh
+git clone git@github.com:aml-org/playground.git
+cd playground
+git checkout develop
+git clone git@github.com:aml-org/playground.git docs
+cd docs
+git checkout gh-pages
+cd ..
+npm install
+gulp bundleAll
+```
+
+### Running examples
+To run a demo, execute corresponding gulp `serve...` command. To get a list of all available `serve...` commands, please see [gulpfile.js](https://github.com/aml-org/playground/blob/master/gulpfile.js).
+
+### Creating pull requests
+Please create source code PRs to branch `develop` and html/css/etc to branch `gh-pages`.
+
+Before creating a PR, make sure to run `gulp bundleAll` so all the code is compiled/compressed. If you've followed [installation instructions](#installation), running `gulp bundleAll` should update both source code and compiled gh-pages code. This will allow you switch to `docs` directory and create a PR to `gh-pages` branch as well.
```

Comment:
what does `switch to docs directory` mean? copy content of `docs/` to a clone/fork of the `gh-pages` branch and then PR?

Reply:
"Installation" section suggests (and previous sentence mentions it) setting up the project so that the `docs/` folder contains `gh-pages` branch content. When user runs `gulp bundleAll` it would compile Typescript code to `docs/js` as well as `docs/scss` to `docs/css`.